tests/test_widget.py

```diff
@@ -317,3 +316,76 @@ def check_dialog_and_click_no(dialog):
         dialog_action=check_dialog_and_click_no,
         time_out=5,
     )
+
+
+def test_radius_save_preserves_old_data(cpt_widget):
+    """Test saving a new particle radius does not mess up the previous one."""
+    # setup measurement layer
+    # layer_measurements = cpt_widget._setup_measurement_layer()
+    layer_measurements = cpt_widget.viewer.add_points(name="Radii and Lengths", ndim=2)
```

Comment:
```suggestion
    measurements_layer = cpt_widget.viewer.add_points(name="Radii and Lengths", ndim=2)
```
I'd actually call it `measurements_layer` for clarity. (Obviosuly if you want to do that, don't just accept this here, but rename the symbol in VS Code.